### PR TITLE
#164771588 create detail view

### PR DESCRIPTION
**Pull request by Maxfurry**
### What does this PR do?
Creates developer detail layout view

### Description of Task to be completed?
* Create activity_dev_details.xml file
* Create activity_main.xml file
* Create content_dev_details.xml file
* Update recycler_item.xml file

### How should this be manually tested?
* Clone the repo
* Checkout to branch ft-create-detailed-activity-layout-164771588
* Open activity_dev_details.xml
* Open the design tab
* Compare the layout file to the screenshot below.

### What are the relevant pivotal tracker stories?
[#164771588](https://www.pivotaltracker.com/story/show/164771588)

### Screenshots(if available)
Screen Shot 2019-03-28 at 3 34 37 PM

### CODE REVIEW

File: app/src/main/java/com/example/javadevnai/DevDetails.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        getMenuInflater().inflate(R.menu.action_dev_details, menu);
+        return super.onCreateOptionsMenu(menu);
+    }
+
+        @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.share:
+
+                Intent sharingIntent = new Intent(android.content.Intent.ACTION_SEND);
```
Comment (bibangamba) [inline app/src/main/java/com/example/javadevnai/DevDetails.java]: remove this implementation of the `share feature` as you're doing work that is outside the scope of the task. just stub it; either with a `toast` or `snackbar` for now

**Review by bibangamba — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/javadevnai/DevDetails.java (hunk 1 above)
Comment (Maxfurry) [reply app/src/main/java/com/example/javadevnai/DevDetails.java]: ok, on it.

Comment (Maxfurry) [reply app/src/main/java/com/example/javadevnai/DevDetails.java]: Completed

File: app/src/main/res/layout/content_dev_details.xml [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    </android.support.constraint.ConstraintLayout>
+
+
+    <android.support.constraint.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="383dp"
+        android:padding="20sp"
+        app:layout_constraintTop_toBottomOf="@+id/constraintLayout">
+
+        <LinearLayout
```
Comment (bibangamba) [inline app/src/main/res/layout/content_dev_details.xml:94]: you're misusing the constraint layout here. ideally, you should use constraints to help you arrange. you're instead putting different layouts inside your constraint layout to handle aligning/positioning; which should be handled by the constraint layout (something similar to the orange background constraint layout above). In other words, remove the LinearLayouts and use constraints directly on relevant elements

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -8,4 +8,13 @@
         <item name="colorAccent">@color/colorAccent</item>
     </style>
 
+    <style name="AppTheme.NoActionBar">
```
Comment (bibangamba) [inline app/src/main/res/values/styles.xml:11]: I don't think you need this right now, why are they part of the PR?

**Review by bibangamba — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (Maxfurry) [reply app/src/main/res/values/styles.xml:11]: It is for the action bar
It was auto generated

File: app/src/main/res/layout/content_dev_details.xml (hunk 2 above)
Comment (Maxfurry) [reply app/src/main/res/layout/content_dev_details.xml:94]: I know that constraint layout removes the necessity to nest layouts into each other but there are somethings linear layout does better than constraint, but the major reason for using it was to group the children.

Comment (Maxfurry) [reply app/src/main/res/layout/content_dev_details.xml:94]: I'll see about using views or Fragment if it will still give me my desired result

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (bibangamba) [reply app/src/main/res/values/styles.xml:11]: ah, okay...🤔

File: app/src/main/res/layout/content_dev_details.xml (hunk 2 above)
Comment (bibangamba) [reply app/src/main/res/layout/content_dev_details.xml:94]: you can use chaining for this... but I won't insist upon it. it would have been better for you to learn and implement that way.

**Review by bibangamba — APPROVED** [review]

File: app/src/main/res/layout/content_dev_details.xml (hunk 2 above)
Comment (Maxfurry) [reply app/src/main/res/layout/content_dev_details.xml:94]: I'll look into chaining then
